packages/certs-v5/lib/display_table.js

```diff
@@ -25,14 +25,22 @@ function type (f) {
 
 module.exports = function (certs) {
   let mapped = certs.filter(function (f) { return f.ssl_cert }).map(function (f) {
-    return {
+    let tableContents = {
       name: f.name,
       cname: f.cname,
       expires_at: f.ssl_cert.expires_at,
       ca_signed: f.ssl_cert['ca_signed?'],
       type: type(f),
-      common_names: f.ssl_cert.cert_domains.join(', ')
+      common_names: f.ssl_cert.cert_domains.join(', '),
     }
+
+    // If they're using ACM it's not really worth showing the number of associated domains since
+    // it'll always be 1 and is entirely outside the user's control
+    if (!f.ssl_cert.acm) {
+      tableContents.associated_domains = (f.domains && f.domains.length) ? f.domains.length : '0'
```

Comment:
I was a bit confused seeing this in the output and after reading the comment. What do you think about using `1` here instead of '0'? Or showing a blank space? 0 makes me think something isn't quite right since there is a cname for the cert. I might also be missing a key difference on the domains field vs the cname field.

Reply:
Ah, yes. The cname field is indicating the domain(s) provided when *creating* the cert. The "associated domains" field is referring to the number of actual connections made between this cert and actual domains that belong to the app on Heroku.

In other words, you may have a cert with a cname that reads "*.purple.com", and in Heroku, that cert could be assigned to "{foo,bar,baz}.purple.com" so you'd have 3 associated domains.